.gitignore

```diff
@@ -6,3 +6,4 @@
 /oracle-sm.iml
 /scTopics
 /.env
+/buildBackups/
```

Comment:
As this is related to your private workflow please add this line to your global `.gitignore` instead.